internal/main.go

```diff
@@ -59,25 +69,45 @@ type TV struct {
 	FirstAirDate  string `json:"first_air_date"`
 	Genres        []struct {
 		Name string `json:"name" gorm:"type:text"`
-	} `json:"genres" gorm:"embedded[];type:text[]"`
-	PosterPath    string  `json:"poster_path"`
-	Popularity    float32 `json:"popularity"`
-	WatchProvider struct {
-		Results map[string]struct {
-			Flatrate []struct {
-				DisplayPriority int    `json:"display_priority"`
-				LogoPath        string `json:"logo_path"`
-				Id              int    `json:"provider_id"`
-				Name            string `json:"provider_name"`
-			} `json:"flatrate"`
-			Free []struct {
-				DisplayPriority int    `json:"display_priority"`
-				LogoPath        string `json:"logo_path"`
-				Id              int    `json:"provider_id"`
-				Name            string `json:"provider_name"`
-			} `json:"free"`
-		} `json:"results" gorm:"column:providers;type:json"`
-	} `json:"watch/providers" gorm:"embedded"`
+	} `json:"genres" gorm:"-"`
+	PosterPath string   `json:"poster_path"`
+	Popularity float32  `json:"popularity"`
+	Providers  Provider `json:"watch/providers"`
+}
+
+func (tv *TV) ToMedia() *DBMedia {
```

Comment:
Is the capital `T` in `ToMedia()` important?

Reply:
Nope, changed it to `toMedia()` :D